components/bl-leafletMap-component/src/lib/leaflet/leaflet-src.js

```diff
@@ -0,0 +1,14397 @@
+/* @preserve
```

Comment:
all lib files must be minified

Reply:
Check please. I used microsoft ajax minifier.

Reply:
Initially, the file was not minified in the library, after Dima's comment, I minified it using the microsoft ajax minifier utility and it works. I'm sorry, I don't understand, is everything normal at the moment or do I need to fix something?